cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -12,10 +12,28 @@ class HttpCache
   # A map from script name to script level URL pattern.
   CACHED_SCRIPTS_MAP = {
```

Comment:
(to be pedantic) it's still a constant but a mutable one, I'll rewrite it to be immutable though.